frontend/src/components/deployments/api-deployment/ApiDeployment.jsx

```diff
@@ -233,6 +234,38 @@ function ApiDeployment() {
       });
   };
 
+  const downloadPostmanCollection = () => {
+    apiDeploymentsApiService
+      .downloadPostmanCollection(selectedRow?.id)
+      .then((res) => {
+        const href = URL.createObjectURL(res?.data);
```

Comment:
@chandrasekharan-zipstack Suggestion: For better code quality, we can destructure the response and use the extracted variables wherever necessary.

`const { data, headers } = res;`